### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by haitien**
ready

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | <%= t("ror_sampleapp")%></title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (baclv) [inline app/views/layouts/application.html.erb]:
```suggestion
    <title><%= full_title yield :title %></title>
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("ror_sampleapp")
```
Comment (baclv) [inline app/helpers/application_helper.rb]:
```suggestion
    base_title = t "ror_sampleapp"
```

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+    page_title.empty? ? base_title : page_title + " | " + base_title
```
Comment (baclv) [inline app/helpers/application_helper.rb]:
```suggestion
    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

**Comment by baclv** [comment]
commented

**Comment by haitien** [comment]
ready

**Comment by thanhhungchu95** [comment]
merged

**Pull request merged**